static/src/js/components/GranuleFilters/GranuleFiltersForm.jsx

```diff
@@ -445,7 +445,7 @@ export const GranuleFiltersForm = (props) => {
                       value: isChecked
                     })
 
-                    // This timeout is needed for there to be no delay in the checkbox working
+                    // Previously there was a timeout here to await the event loop
```

Comment:
Do we need this comment anymore?

Reply:
Ehh I would rather keep this because, I cannot explain why we used to have a setTimeout there prior to the granule filter update I made. If you really want I will remove it

Reply:
Its just odd to comment code that isn't there anymore. We should not be making changes we don't feel comfortable with. Leaving something like this behind feels like a red flag.